microsite/data/plugins/pager-duty.yaml

```diff
@@ -0,0 +1,13 @@
+---
+title: PageDuty
+author: Spotify
+authorUrl: https://github.com/spotify
+category: Discovery
```

Comment:
Not sure that discovery is the correct category here.

How about Monitoring for now

Reply:
@benjdlambert updated!